misc/prime_seive.c

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief [Prime Seive](https://leetcode.com/problems/count-primes/)
+ * algorithm implementation.
+ * @author[Divyansh Kushwaha](https://github.com/webdesignbydivyansh)
+ */
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Add a brief description of what the libraries/headers add (see the example below).
```suggestion
#include <assert.h>      /// for assert
#include <stdio.h>      /// for
#include <stdlib.h>    /// for
```